### PR TITLE
Executor and databox tests

### PR DESCRIPTION
**Pull request by RjiukYagami**
Few changes in databox and executor because there were some problems that came up in tests.

### CODE REVIEW

File: app/databox/databox.py [hunk 1]
```diff
@@ -63,6 +63,7 @@ def is_builtin(command_id):
 
 _USER_DEFINED_COMMANDS = {}
 _BUILTIN_COMMANDS = {
+    '0': Command('echo', 'test'),
```
Comment (0mp) [inline app/databox/databox.py]: Why is it here?

File: app/executor/executor.py [hunk 2]
```diff
@@ -27,12 +27,13 @@ def execute(command_id):
         otherwise.
     """
     command_id = str(command_id)
-    command, arguments = databox.get_command_and_arguments(command_id)
-    if command is None:
+    gasp = databox.get_command_and_arguments(command_id)
```
Comment (0mp) [inline app/executor/executor.py]: What is it?

File: app/databox/databox.py (hunk 1 above)
Comment (RjiukYagami) [inline app/databox/databox.py]: So i do not invoke browser during test.

File: app/executor/executor.py (hunk 2 above)
Comment (RjiukYagami) [inline app/executor/executor.py]: when function cannot find command it returns None, but we wanted two returns and it raised error.

File: app/databox/databox.py (hunk 1 above)
Comment (0mp) [inline app/databox/databox.py]: k

File: app/executor/executor.py (hunk 2 above)
Comment (0mp) [inline app/executor/executor.py]: ok

**Comment by 0mp** [comment]
ready to merge :)

**Pull request merged**